package.json

```diff
@@ -78,5 +78,10 @@
   },
   "dependencies": {
     "http-server": "14.1.1"
+  },
+  "pnpm": {
```

Comment:
Hoewel deze patch specifiek is voor de package `proprietary/design-tokens` dwingt `pnpm` af dat dit in de project root wordt geplaatst